configure.ps1

```diff
@@ -138,6 +138,7 @@ $CouchDBConfig = @"
 {node_name, "-name couchdb@localhost"}.
 {cluster_port, 5984}.
 {backend_port, 5986}.
+{prometheus_port, 17986}.
```

Comment:
stray commit?

Reply:
No, I only adapted this to the *nix configure-[version](https://github.com/apache/couchdb/blob/33ee2164e6e73429193bc353a2c7dfb9d27a7790/configure#L261)

Reply:
ok, but this also has nothing to do with SM91? It’d be good to keep topical things in separate commits/PRs

Reply:
ok, fixed. Should the doc update be separate too?